src/OpenTelemetry.Exporter.Stackdriver/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 * Update OpenTelemetry SDK version to `1.8.0`.
   ([#1635](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1635))
+* Fixes an issue when Activity/ActivityLink tags contain duplicate tag keys
```

Comment:
> Is user doing `AddTag` instead of `SetTag` ?

When I encountered the issue, that's what I was doing, yes.

Reply:
I see. Maybe we need to improve the doc on the upstream to prevent this issue from occurring, instead of fixing each exporter. (for example, OTLP Exporter does not handle deduplication, as it expects upstream to handle it.)

Reply:
Not opposed to this PR anyway. 
The doc part is something needed iirespective.